vm/nova/vm.go

```diff
@@ -512,6 +452,10 @@ func accountStakingSTVF(vmParams *vm.Params, chainID iotago.ChainID, current *io
 	_, isClaiming := vmParams.WorkingSet.Rewards[chainID]
 
 	if currentStakingFeat != nil {
+		if next.FeatureSet().BlockIssuer() == nil {
```

Comment:
What if the staking feature was removed here? Then it would  be ok for next block issuer feature to be removed too, right? This is both state and governance transition now, so one transaction could remove both block issuer feature and staking feature I think.